src/view/adminhtml/web/js/filter-attributes.js

```diff
@@ -86,6 +102,11 @@ define([
                     }
                 });
             });
+
+            //select diffrent filter template
+            $('select[name="tweakwise_filter_template"]').on('change', function(evt) {
+                $('select[name*="filter_attributes"]').trigger('initAttributes');
```

Comment:
When adding new rows to the Filter Attributes, it will bind this event. But it seems that they don't get cleaned up. When you delete a Filter Attribute row, and then change the filter template, it will do an ajax request for each row that was previously added.

This causes a lot of unnecessary ajax requests. There should probably be some kind of event listener cleanup